### ISSUE TITLE
Improve Deep Links

### ISSUE THREAD

**Issue by co-in** [dashevo/dash-wallet#647]
At now we have to use two links: 

One for the application
**https://play.google.com/store/apps/details?id=hashengineering.darkcoin.wallet&launch=true**

Second for accepting payment
**dash:Xgzpd6doencq7ihrz9nvDrYEUaRTfDKm2Q?amount=1**

It would be very cool to post a link to download and accept payment in one QR, for install app and pay
**https://play.google.com/store/apps/details?id=hashengineering.darkcoin.wallet&launch=true&pay=Xgzpd6doencq7ihrz9nvDrYEUaRTfDKm2Q&amount=1**

If I'm not mistaken, this is done somehow through **android:sspPrefix** or **android:sspPattern**
```
<data 
  android:scheme="https" 
  android:host="play.google.com" 
  android:sspPrefix="/store/apps/details?id=hashengineering.darkcoin.wallet"
/>
```

**Comment by HashEngineering** [comment]
This is an interesting idea.  However, if the user needs to download Dash Wallet from the Play Store, that means that this user doesn't have any Dash to make a payment.

Otherwise, I suppose this smarter deep link would work to start only Dash Wallet when payment is requested as opposed to other coin wallets that support dash.  If multiple apps on a device support Dash, then android will bring up a list of all of these apps and even allow the user to use one by default.

**Comment by co-in** [comment]
That's right, even if the user does not have funds in the wallet, he will at least install the application, then deal with it, try it, and pay next time. We get a high bounce rate for new users, as most of them do not understand or even know what DASH is.

Steve Jobs would not approve of using other apps)

**Comment by HashEngineering** [comment]
There is another way to do this (go to google play to install) using dynamic links, but this service is being discontinued by Google in 2025.  There may be alternatives.
